corpus/source_files.txt

```diff
@@ -55,6 +55,33 @@ Line comments
 (source_file
   (line_comment))
 
+============================================
+Doc comments
+============================================
+
+/// Doc
+/// Comment
```

Comment:
`//!` as well.

But I think `//!!` will become normal comment. The last I worked on is this

https://github.com/mawww/kakoune/blob/871782faaf954cda654cdbdcb3590b45534607d1/rc/filetype/rust.kak#L46-L49

Reply:
From https://doc.rust-lang.org/reference/comments.html#examples

> //!  - Inner line doc
> //!! - Still an inner line doc (but with a bang at the beginning)

So I implemented it as "any ! makes it a doc comment, no matter how many"

Reply:
From https://doc.rust-lang.org/reference/comments.html#examples

> //!  - Inner line doc
> //!! - Still an inner line doc (but with a bang at the beginning)

So I implemented it as "any ! makes it a doc comment, no matter how many"

Reply:
I believe it's only up till four, IIRC it's the same highlight in vim.

Reply:
In SpaceVim (without LSP) the highlight is different (grey for comments, orange for doc comments) and that was really useful IMHO.